lib/workers/repository/result.ts

```diff
@@ -19,7 +20,13 @@ import {
 } from '../../constants/error-messages';
 import { logger } from '../../logger';
 
-type ProcessStatus = 'disabled' | 'enabled' | 'onboarding' | 'unknown';
+type ProcessStatus =
```

Comment:
```suggestion
export type ProcessStatus =
```

not your fault, but it's used in public interface, so should also be exported 😉